controllers/job.go

```diff
@@ -180,21 +181,25 @@ func (r *OSArtifactReconciler) newBuilderPod(pvcName string, artifact *osbuilder
 		})
 	}
 
-	cloudImgCmd := fmt.Sprintf(
-		"/raw-images.sh /rootfs /artifacts/%s.raw",
-		artifact.Name,
-	)
+	var cloudImgCmd strings.Builder
+	cloudImgCmd.WriteString("auroraboot --debug")
+	cloudImgCmd.WriteString(" --set 'disk.raw=true'")
+	cloudImgCmd.WriteString(" --set 'disable_netboot=true'")
+	cloudImgCmd.WriteString(" --set 'disable_http_server=true'")
+	cloudImgCmd.WriteString(" --set 'state_dir=/artifacts'")
+	cloudImgCmd.WriteString(" --set 'container_image=dir:/rootfs'")
 
 	if artifact.Spec.CloudConfigRef != nil {
 		volumeMounts = append(volumeMounts, corev1.VolumeMount{
 			Name:      "cloudconfig",
-			MountPath: "/iso/iso-overlay/cloud_config.yaml",
+			MountPath: "/cloud-config.yaml",
 			SubPath:   artifact.Spec.CloudConfigRef.Key,
 		})
-
-		cloudImgCmd += " /iso/iso-overlay/cloud_config.yaml"
+		cloudImgCmd.WriteString(" --cloud-config /cloud-config.yaml")
 	}
 
+	cloudImgCmd.WriteString(fmt.Sprintf(" && mv /artifacts/*.raw /artifacts/%s.raw", artifact.Name))
+
 	if artifact.Spec.CloudConfigRef != nil || artifact.Spec.GRUBConfig != "" {
 		cmd = fmt.Sprintf(
 			"auroraboot --debug build-iso --override-name %s --date=false --overlay-iso /iso/iso-overlay --output /artifacts dir:/rootfs",
```

Comment:
Does `--overlay-iso` flag need to change considering the cloud-config mount has changed in https://github.com/kairos-io/osbuilder/pull/201/files#diff-a31896bf5eb286732689977ee4604b0acc2f05db5524472c910904ccde7b9da9R195?

Reply:
Getting following error:

```
+ auroraboot --debug build-iso --override-name hello-kairos --date=false --overlay-iso /iso/iso-overlay --output /artifacts dir:/rootfs
invalid path '/iso/iso-overlay'
```

Reply:
Actually should this be changed to `--cloud-config`
```suggestion
			"auroraboot --debug build-iso --override-name %s --date=false --cloud-config /cloud-config.yaml --output /artifacts dir:/rootfs",
```

Reply:
good catch!

Reply:
I refactored this to use a string builder like the rest of them.